tests/puppeteer/pages/BO/catalog/brands/index.js

```diff
@@ -14,59 +14,55 @@ module.exports = class Brands extends BOBasePage {
-    this.sortColumnSpanButton = `${this.sortColumnDiv} span.ps-sort`;
+    this.tableHead = table => `${this.gridTable(table)} thead`;
+    this.sortColumnDiv = (table, column) => `${this.tableHead(table)
+    } div.ps-sortable-column[data-sort-col-name='${column}']`;
+    this.sortColumnSpanButton = (table, column) => `${this.sortColumnDiv(table, column)} span.ps-sort`;
 
     // Grid Actions
-    this.gridActionButton = '#%TABLE-grid-actions-button';
-    this.gridActionDropDownMenu = 'div.dropdown-menu[aria-labelledby=\'%TABLE-grid-actions-button\']';
-    this.gridActionExportLink = `${this.gridActionDropDownMenu} a[href*='/export']`;
+    this.gridActionButton = table => `#${table}-grid-actions-button`;
+    this.gridActionDropDownMenu = table => `div.dropdown-menu[aria-labelledby='${table}-grid-actions-button']`;
+    this.gridActionExportLink = table => `${this.gridActionDropDownMenu(table)} a[href*='/export']`;
 
     // Brands list Selectors
-    this.brandsTableColumnLogoImg = `${this.tableColumn
-      .replace('%TABLE', 'manufacturer').replace('%COLUMN', 'logo')} img`;
-    this.brandsTableEnableColumn = `${this.tableColumn
-      .replace('%TABLE', 'manufacturer').replace('%COLUMN', 'active')}`;
-    this.brandsEnableColumnValidIcon = `${this.brandsTableEnableColumn} i.grid-toggler-icon-valid`;
-    this.brandsEnableColumnNotValidIcon = `${this.brandsTableEnableColumn} i.grid-toggler-icon-not-valid`;
-    this.viewBrandLink = `${this.actionsColumn} a[data-original-title='View']`
-      .replace('%TABLE', 'manufacturer');
-    this.editBrandLink = `${this.dropdownToggleMenu} a[href*='/edit']`.replace('%TABLE', 'manufacturer');
-    this.bulkActionsEnableButton = `${this.gridPanel} #manufacturer_grid_bulk_action_enable_selection`
-      .replace('%TABLE', 'manufacturer');
-    this.bulkActionsDisableButton = `${this.gridPanel} #manufacturer_grid_bulk_action_disable_selection`
-      .replace('%TABLE', 'manufacturer');
-    this.deleteBrandsButton = `${this.gridPanel} #manufacturer_grid_bulk_action_delete_selection`
-      .replace('%TABLE', 'manufacturer');
+    this.brandsTableColumnLogoImg = row => `${this.tableColumn('manufacturer', row, 'logo')} img`;
+    this.brandsTableEnableColumn = row => `${this.tableColumn('manufacturer', row, 'active')}`;
+    this.brandsEnableColumnValidIcon = row => `${this.brandsTableEnableColumn(row)} i.grid-toggler-icon-valid`;
+    this.brandsEnableColumnNotValidIcon = row => `${this.brandsTableEnableColumn(row)} i.grid-toggler-icon-not-valid`;
+    this.viewBrandLink = row => `${this.actionsColumn('manufacturer', row)} a[data-original-title='View']`;
+    this.editBrandLink = row => `${this.dropdownToggleMenu('manufacturer', row)} a[href*='/edit']`;
+    this.bulkActionsEnableButton = `${this.gridPanel('manufacturer')} #manufacturer_grid_bulk_action_enable_selection`;
+    this.bulkActionsDisableButton = `${this.gridPanel('manufacturer')
+    } #manufacturer_grid_bulk_action_disable_selection`;
```

Comment:
Why this end of line ?

Reply:
Linter did this...

Reply:
Bad linter.

Reply:
Isn't a rule with length of lines ? that could be adjusted

Reply:
we already adjust it to 120 characters, i think it's enough.